lib/sqlalchemy/testing/reflection_fixture.py

```diff
@@ -0,0 +1,201 @@
+from . import config
+from .assertions import eq_
+from .assertions import is_
+from .assertions import is_not_
+from .assertions import is_true
+from .fixtures import TablesTest
+from .. import inspect
+from .. import Integer
+from ..schema import Column
+from ..schema import Computed
+from ..schema import MetaData
+from ..schema import Table
+
+
+class ComputedReflectionFixtureTest(TablesTest):
+    run_inserts = run_deletes = None
+
+    __backend__ = True
+    __requires__ = ("computed_columns", "table_reflection")
+
+    return_persisted = None
+    default_persisted = None
+    support_stored = None
+    support_virtual = None
+
+    def to_sqltext(self, column, op, value):
+        assert False, "override in dialect"
+
+    @classmethod
+    def _col(cls, name, sqltext, persisted):
+        if (
+            persisted
+            and cls.support_stored
+            or not persisted
+            and cls.support_virtual
+        ):
+            return [
+                Column(name, Integer, Computed(sqltext, persisted=persisted))
+            ]
+        return []
+
+    @classmethod
+    def define_tables(cls, metadata):
+        Table(
+            "computed_default_table",
+            metadata,
+            Column("id", Integer, primary_key=True),
+            Column("normal", Integer),
+            Column("computed_col", Integer, Computed("normal + 42")),
+            Column("with_default", Integer, server_default="42"),
+        )
+
+        Table(
+            "computed_column_table",
+            metadata,
+            Column("id", Integer, primary_key=True),
+            Column("normal", Integer),
+            Column("computed_no_flag", Integer, Computed("normal + 42")),
+            *cls._col("computed_virtual", "normal + 2", False),
```

Comment:
python 2 won't accept the multiple *, so I think the approach should be, "if testing.requirements.computed_columns_stored.enabled(): table.append_column(Column(...))"